beacon-chain/p2p/encoder/ssz_test.go

```diff
@@ -19,10 +20,13 @@ func TestSszNetworkEncoder_RoundTrip(t *testing.T) {
 	testRoundTripWithGossip(t, e)
 }
 
-func TestSszNetworkEncoder_RoundTrip_Snappy(t *testing.T) {
```

Comment:
Note: I deleted this old test, it was identical to TestSszNetworkEncoder_RoundTrip